src/js/components/FormField/FormField.js

```diff
@@ -334,7 +336,10 @@ const FormField = forwardRef(
       borderColor = (themeBorder && themeBorder.color) || 'border';
     }
 
-    const labelStyle = { ...formFieldTheme.label };
+    let labelStyle;
+    if (formKind === 'survey') {
```

Comment:
Do we want kind === 'survey' to be hardcoded or do we want something a bit more loose like how button kind works?

Reply:
aw okay so if `(kind)` then do styles rather then `=== survey`

Reply:
I think it might be nice to have a little more flexibility here like how button handles kinds. This would allow people to have 'marketing' forms or other styling variations we haven't thought about

Reply:
yes changed this to be whatever `kindArg` they pass